src/services/parseFilesAsAttachments.ts

```
@@ -77,6 +115,17 @@ export async function parseFilesAsAttachmentsLegacy(
   return attachments
 }
 
-export default function parseFilesAsAttachments() {
-  // TODO: Rotate blob files for new stoarge types
+export async function correctNewAttachmentOrientation(
+  attachment: AttachmentNew,
+): Promise<AttachmentNew> {
+  try {
+    const blob = await correctFileOrientationAsBlob(attachment.data)
+    return {
+      ...attachment,
+      data: blob,
+    }
+  } catch (err) {
+    console.warn('Failed to rotate the orientation of a file:', err)
+    return attachment
```

If we are ignoring this error, prob want to send it to Sentry.